src/main/java/mods/eln/misc/VoltageLevelColor.kt

```diff
@@ -40,6 +41,8 @@ enum class VoltageLevelColor(private val voltageLevel: String?) {
                 HighVoltage
             } else if (voltage <= 2 * Eln.VVU) {
                 VeryHighVoltage
+            } else if (voltage <= 2 * Eln.CCU) {
+                Neutral
```

Comment:
I was thinking of not bothering with the color underlay for these items but perhaps it makes sense?